arangod/IResearch/IResearchRocksDBLink.cpp

```diff
@@ -44,7 +44,8 @@ namespace iresearch {
 
 IResearchRocksDBLink::IResearchRocksDBLink(IndexId iid, LogicalCollection& collection)
     : RocksDBIndex(iid, collection, IResearchLinkHelper::emptyIndexSlice(),
-                   RocksDBColumnFamily::invalid(), false),
+                   RocksDBColumnFamilyManager::get(RocksDBColumnFamilyManager::Family::Invalid),
```

Comment:
I would prefer to have explicit getters like before, even if they internally just forward to `get(<Family>)` because I find it easier to read, but no strong opinion here.

Reply:
I think the names used before aren't very clear. What I mean is `invalid()` doesn't tell me it's a getter, where as `get(Family::Invalid)` is self-describing. I guess we could do something like `getFamilyInvalid()` but personally I'd rather keep it generic and have people engage with the enum, as that's standardized across different interfaces (including also `set`, and `RocksDBOptionFeature::columnFamilyOptions).

Reply:
Fine by me. :-)